app/Controllers/tagController.php

```diff
@@ -152,11 +152,15 @@ public function addAction(): void {
 		$name = Minz_Request::paramString('name');
 		$tagDAO = FreshRSS_Factory::createTagDao();
 		if (strlen($name) > 0 && null === $tagDAO->searchByName($name)) {
-			$tagDAO->addTag(['name' => $name]);
-			Minz_Request::good(_t('feedback.tag.created', $name), ['c' => 'tag', 'a' => 'index']);
+			$id_tag = $tagDAO->addTag(['name' => $name]);
+			if ($id_tag !== false) {
+				Minz_Request::good(_t('feedback.tag.created', $name), ['c' => 'tag', 'a' => 'index']);
+			} else {
+				Minz_Request::bad(_t('Label cannot have the same name as Category', $name), ['c' => 'tag', 'a' => 'index']);
```

Comment:
This should be a new i18n string. Can be done semi-automatically with a command like:

```sh
cli/manipulate.translation.php -a add -k 'feedback.tag.category_exists' -v 'Label cannot have the same name as a category.'
```